job/job.go

```diff
@@ -174,12 +174,17 @@ func (j *Job) requirements() map[string][]string {
 			requirements[key] = make([]string, 0)
 		}
 
-		if uni != nil {
-			for i, v := range values {
-				values[i] = unitPrintf(v, *uni)
+		processedValues := make([]string, len(values))
+
+		for i, v := range values {
```

Comment:
@kayrus thanks for the catch! you can put this inside the same previous loop:
`if uni != nil { processedvalues := make()
     for ... { ...}
     requirements[key] = processedvalues
} else { requirements[key] = values } `

Otherwise it's ok too!